.storybook/themes/news.scss

```diff
@@ -976,6 +985,15 @@
 						),
 					),
 				),
+				links-bar: (
+					display: flex,
```

Comment:
This is doing the same as the `<Stack>` component - can we use the `<Stack>` ?

Reply:
Also these styles are duplication of the styles set above - you only need to redefine the styles here **if** there are changes

Reply:
this looks like it's been resolved